app/main/forms/auth_forms.py

```diff
@@ -1,38 +1,10 @@
 from flask_wtf import Form
-from wtforms import PasswordField
-from wtforms.validators import DataRequired, Regexp, EqualTo, Length
+from wtforms.validators import DataRequired
 
-from dmutils.forms import StripWhitespaceStringField
-
-
-class LoginForm(Form):
-    email_address = StripWhitespaceStringField('Email address', validators=[
-        DataRequired(message="Email address must be provided"),
-        Regexp("^[^@^\s]+@[^@^\.^\s]+(\.[^@^\.^\s]+)+$",
-               message="Please enter a valid email address")
-    ])
-    password = PasswordField('Password', validators=[
-        DataRequired(message="Please enter your password")
-    ])
+from dmutils.forms import EmailField
 
 
 class EmailAddressForm(Form):
-    email_address = StripWhitespaceStringField('Email address', validators=[
-        DataRequired(message="Email address must be provided"),
-        Regexp("^[^@^\s]+@[^@^\.^\s]+(\.[^@^\.^\s]+)+$",
-               message="Please enter a valid email address")
-    ])
-
-
-class ChangePasswordForm(Form):
-    password = PasswordField('Password', validators=[
-        DataRequired(message="Please enter a new password"),
-        Length(min=10,
-               max=50,
-               message="Passwords must be between 10 and 50 characters"
-               )
-    ])
-    confirm_password = PasswordField('Confirm password', validators=[
-        DataRequired(message="Please confirm your new password"),
-        EqualTo('password', message="The passwords you entered do not match")
+    email_address = EmailField('Email address', validators=[
+        DataRequired(message="Email address must be provided")
```

Comment:
I think part of this tech debt ticket was around validating length in more places. It feels like maybe we should add some length validation on email addresses (256 is probably fine, but we could be more generous at 512 if there feels like any good reason...). Not sure if it's better to just add it here or in dmutils though.

Reply:
Yeah, I guess I was thinking the email regex would handle it, but I guess it doesn't stop people chucking really long but still valid email addresses in.

This suggests email addresses should have a limit of 254 chars. I'm not sure this includes if you use the '+...@' trick though, as I just sent an email with a 324 character long address to myself.

Either way, I think 512 should be fine. And I do think it should go in the utils. Will add their and link the PR here.

Reply:
https://github.com/alphagov/digitalmarketplace-utils/pull/372